docs/data/joy/getting-started/support/support.md

```diff
@@ -27,17 +27,22 @@ We require bug reports to be accompanied by a **minimal reproduction**.
 It significantly increases the odds of fixing the problem.
 You have a few possible options to provide it:
 
-- You can browse the documentation, find an example close to your use case, and then open it in a live editor:
-  <a href="/joy-ui/react-button/#basics">
-  <span class="only-light-mode">
-  <img src="/static/docs-infra/forking-an-example.png" alt="Forking an example" loading="lazy" width="1548" height="606" style="display: block; max-width: 774px;">
```

Comment:
Cool change, but I think we should propagate those everwhere. Handled for Base UI, Material UI, MUI System, and MUI X.